### PR TITLE
Au qa 100 alarm9 group dead sensor alarm

### PR DESCRIPTION
**Pull request by GreeshmaPK**
Tracker updated.

### CODE REVIEW

File: Resources/apiresources.robot [hunk 1]
```diff
@@ -923,3 +923,13 @@ checkAlarmStatusForAllAHUsInGroup
         #log to console  ${json_dictionary["data"]}
         log to console  ==================${alarm_type} Alarm Cleared for ahu ${ahu_list}[${i}]====================
     END
+
+    #Created by Greeshma on 28th Dec 2021
+overrideAllAHUsWithBOPValueON
```
Comment (ananiac) [inline Resources/apiresources.robot:928]: overrideAllAHUsWithBOPValueOFF can be parameterized to pass the value 1or 0

**Review by ananiac — COMMENTED** [review]
Additionally minOnTestResources.robot
keyword minOnTestOnGeneralTestPrecondition
comment     ...                Also write test entry temperature for the parallel staleStatePrevention program  can be removed as stalestate is not used

File: Resources/apiresources.robot (hunk 1 above)
Comment (GreeshmaPK) [reply Resources/apiresources.robot:928]: added comment

File: Testcases/AlarmTests/groupColdAlarmTest.robot [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Resource    ${EXECDIR}/Resources/uiresources.robot
+Resource    ${EXECDIR}/Resources/connection.robot
+Resource    ${EXECDIR}/Resources/AlarmTestsResources/alarmResources.robot
+Resource    ${EXECDIR}/Inputs/testInputs.robot
+Resource    ${EXECDIR}/Resources/common.robot
+
+
+*** Test Cases ***
+GroupColdAlarmTest
+    #Steps
```
Comment (ananiac) [inline Testcases/AlarmTests/groupColdAlarmTest.robot:16]: remove #Steps

**Pull request merged**